pages/about.html

```diff
@@ -1,12 +1,66 @@
+
+            <div class="grid-about-content">
+                <p class="about-page-bio">My name is Marjan Abdollahi, my pronouns are she/her. I studied accounting, before changing my career in software development, I worked in the Bank and also for a few years I was a stock trader. 
+                    <p class="about-page-bio">I decided to make the transition into software development because during my trading time I got curious about coding on python which would make trading much easier and disciplined.
+                      So I started with python and in one month my curiosity turned to enjoy. everyday I was searching for some resources to learn new stuffs about coding.
+                    </p>
+                    <p class="about-page-bio">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+                        Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.Lorem ipsum dolor
+                        sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
+                        dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+                    <p class="about-page-bio">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+                        Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.Lorem ipsum dolor
+                        sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
+                        dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+            </div>
+
+            <div class="about-li-container">
+                <ul class="about-ul">
+                    <li class="linkedin" class="about-li"><a class="about-link" href="https://github.com/Marjan-abm">GitHub</a></li>
```

Comment:
To add multiple classes to a tag combine them in one string separated by a space:
```suggestion
                    <li class="linkedin about-li"><a class="about-link" href="https://github.com/Marjan-abm">GitHub</a></li>
```